app/models/repository.rb

```diff
@@ -0,0 +1,20 @@
+class Repository
+  include ActiveModel::Model
+  attr_accessor :title,
+                :link,
+                :description,
+                :created_at,
+                :pull_requests
+
+  def sorted_pull_requests
```

Comment:
Although, I'm not even sure I need it since github returns it to you this way... Still, probably a good thing to declare explicitly